### ISSUE TITLE
Aquementas quest issues

### ISSUE THREAD

**Issue by lphua** [AeroScripts/QuestieDev#1168]
## Description

Issue 1)
No location on Tanaris for where to summon him , 70,49 in Tanaris.  Doesn't require a drop or anything.  Just need to walk to that spot, read the Book of Aquor (11169) , and you get the Silver Totem of Aquementas (11522), and you are done.  You can kill Aquementas or not, doesn't matter.

Issue 2)
After that -- the question mark never pops up on J.D. Collie (9117)'s head

## Wowhead link
https://classic.wowhead.com/quest=4005/aquementas

## Questie version
4.1.1

This one bothering me, I tried to fix look at the lua to try to do a PR for you, but can't find anything wrong with why completion wouldn't show up?

 ```
[11522] = {"Silver Totem of Aquementas",{4005},{},{148507}},
 
[4005] = 
{"Aquementas",
{{9116,},nil,nil,}
,{{9117,},nil,}
,47
,54
,255
,nil
,{"Take the Book of Aquor, the Silvery Claws, and the Irontree Heart to Tanaris.","Use the Book of Aquor to summon Aquementas.","After you have the completed totem, take it to J.D. Collie in Marshal's Refuge.",}
,nil
,{nil,nil,{{11522,nil},},nil,}
,11617
,nil
,{4084,}
,nil
,nil
,nil
,490
,nil
,nil
,nil
,nil
,3961
,nil
,nil
,nil,},
```

**Comment by BreakBB** [comment]
The reason the completion mark doesn't show up is, because the required item (11522) hasn't been in our DB.
But this issue has already been fixed with e5f4305f981bba402f944d7631bd820fed1e9b3c

**Comment by lphua** [comment]
Thanks -- just curious understanding your process:
-You run a script which updates the DB with corrections.lua
-corrections.lua will be wiped clean
?
